app/page.tsx

```diff
@@ -29,6 +31,12 @@ const Index: FC = () => {
     setKeys(generatedKeys);
   }
 
+  function handleLogin(secretKey:string):void {
```

Comment:
This needs to be formatted.
```suggestion
  function handleLogin(secretKey:string): void {
```

Reply:
I have added blanks in this line